source/ios/tools/AdaptiveCards.podspec

```diff
@@ -1,20 +1,25 @@
-  
 Pod::Spec.new do |spec|
   spec.name             = 'AdaptiveCards'
 
-  spec.version          = '2.8.2'
-  
+  spec.version          = '2.8.3'
+
   spec.license          = { :type => 'Adaptive Cards Binary EULA', :file => 'source/EULA-Non-Windows.txt' } 
-  
+
   spec.homepage         = 'https://adaptivecards.io'
-  
+
   spec.authors          = { 'AdaptiveCards' => 'Joseph.Woo@microsoft.com' }
-  
+
   spec.summary          = 'Adaptive Cards are a new way for developers to exchange card content in a common and consistent way'
   
-  spec.source       = { :git => 'https://github.com/microsoft/AdaptiveCards.git', :tag => 'ios-v2.8.2' }
+  spec.source       = { :git => 'https://github.com/microsoft/AdaptiveCards.git', :tag => 'ios-v2.8.3' }
 
-  spec.source_files           = 'source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/*.{h,m,mm}'
+  spec.default_subspecs = 'AdaptiveCardsCore', 'AdaptiveCardsPrivate', 'ObjectModel', 'UIProviders'
+
+  spec.subspec 'AdaptiveCardsCore' do | sspec |
```

Comment:
>sspec [](http://example.com/codeflow?start=40&length=5)

what does sspec stand for, is it short for subspec?

Reply:
correct